src/test/java/Homework16.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void search () {
+       /*
+        1. Create a new file Homework16.java file with @Test annotation in IntelliJ IDEA
+        2. Create a test case createNewPlaylist() using @Test annotation
+        3. Navigate to "https://bbb.testpro.io/"
+        4. Log in with your credentials
+        5. Create a new playlist (use XPath locators to locate the elements)
+        6. Validate that there is a new playlist made
+        7. Create a new branch and commit your changes
+        8. Push your code to a remote repository
+        9. Create a pull request
+        10. Copy and paste the link of the pull request (or your branch) to the field below, so we can check your homework*/
+
+        // xpath to get to element to '+' icon to get options for create new playlist -->  //h1/i[@role="button"]
+        // xpath to get to element to select option for 'New Playlist' --> //li[contains(text(), 'New Playlist')]
+        // xpath to get to element to enter new playlist name --> //form/input[@name]
+        // xpath to get to newly created playlist element --> //li/a[contains(text(), 'Joe Playlist')]
+
+        String myPlaylistName = "Joe Playlist";
+
+        try{
+            login();
+
+            WebElement createNewPlaylistOptions = driver.findElement(By.xpath("//h1/i[@role=\"button\"]"));
+            createNewPlaylistOptions.click();
+            WebElement newPlaylistOption = driver.findElement(By.xpath("//li[contains(text(), 'New Playlist')]"));
+            newPlaylistOption.click();
+            WebElement newPlaylistForm = driver.findElement(By.xpath("//form/input[@name]"));
+            newPlaylistForm.sendKeys(myPlaylistName);
+            newPlaylistForm.sendKeys(Keys.RETURN);
+
+
+            WebElement myPlaylistElement = driver.findElement(
+                    By.xpath("//li/a[contains(text(), '" + myPlaylistName + "')]"));
+            String matchText = myPlaylistElement.getText();
+
+            Assert.assertTrue(matchText.contains(myPlaylistName));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200162618-f8878df3-ddb9-4dd5-b77d-92aef0abf3ea.png)

if your problem is still not solved, you need a DOM-level solution. So, we need an interface that helps to execute JavaScript with Selenium WebDriver. It is called JavaScriptExecutor. JavaScript works at the DOM level. Therefore, we are able to click any web element by using JavaScript executor.

For, example

    ```
   ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", (createNewPlaylistOptions));
    Thread.sleep(2000);
    ((JavascriptExecutor) driver).executeScript("arguments[0].click();", (createNewPlaylistOptions));
    ```
Clicking Elements in Selenium using JavaScript (https://www.baeldung.com/java-selenium-javascript)

![image](https://user-images.githubusercontent.com/104761482/200162952-7a23916d-a200-4ab3-b082-d2bc2415597c.png)

![image](https://user-images.githubusercontent.com/104761482/200163003-bf4a88a4-6b50-4001-adae-6a568a93ebf3.png)